.build/CMakeFiles/3.12.3/CMakeSystem.cmake

```diff
@@ -0,0 +1,15 @@
+set(CMAKE_HOST_SYSTEM "Windows-10.0.14393")
```

Comment:
feel free to add the `.build` directory to `.gitignore` if that's a thing that VS usually has in place.

Reply:
Oops, thx for pointing out.

Reply:
That was actually not VS, just me mimicking the usual directory structure.